nextcord/state.py

```diff
@@ -2307,7 +2307,7 @@ def parse_auto_moderation_action_execution(self, data) -> None:
 
     def parse_guild_audit_log_entry_create(self, data) -> None:
         guild = self._get_guild(int(data["guild_id"]))
-        user_id = int(data["user_id"])
+        user_id = int(data["user_id"]) if data["user_id"] != None else data["user_id"]
```

Comment:
```suggestion
        user_id = int(data["user_id"]) if data["user_id"] is not None else data["user_id"]
```
As `is`/`is not` is used when making comparisons to classes (`None` in this case)